PowerEditor/src/WinControls/FunctionList/functionParser.cpp

```diff
@@ -598,14 +598,14 @@ void FunctionParser::getInvertZones(vector< pair<int, int> > &  destZones, vecto
 {
 	if (sourceZones.size() == 0)
 	{
-		destZones.push_back(pair<int, int>(begin, end));

```

Comment:
Two things:

- `std::make_pair(begin, end)` is more concise.
- Why the C-style casts? Why not `static_cast<int>(begin), static_cast<int>(end)` *more on this in a second*